doc/active_docs/Account Management API.json

```diff
@@ -7506,6 +7506,36 @@
         }
       ]
     },
+    {
+      "path": "/admin/api/services/{service_id}/proxy/deploy.xml",
+      "responseClass": "proxy",
+      "operations": [
+        {
+          "httpMethod": "POST",
+          "summary": "Proxy Deploy",
+          "description": "This will create a new APIcast configuration version for the Staging (and Production in case of Service Mesh) environment with the updated settings.",
```

Comment:
```suggestion
          "description": "Creates a new APIcast configuration version for the Staging environment (production environment in case of Service Mesh)",
```

Reply:
or

```suggestion
          "description": "Promotes the APIcast configuration to the Staging Environment (Production Environment in case of Service Mesh).",
```

Reply:
last one is more in line with ui. you pick.